src/askui/tools/android/agent_os.py

```diff
@@ -210,6 +210,39 @@ def __repr__(self) -> str:
             f"display_name={self.display_name}, display_id={self.display_id})"
         )
 
+    def get_display_id_flag(self) -> str:
+        """
+        Returns the display ID flag for shell commands.
+
+        Returns:
+            str: The display ID flag in the format `-d {display_id}`.
+        """
+        return f"-d {self.display_id}"
+
+    def get_display_unique_id_flag(self) -> str:
+        """
+        Returns the display unique ID flag for shell screencap command.
+
+        Returns:
+            str: The display unique ID flag in the format `-d {unique_display_id}`.
+        """
+        return f"-d {self.unique_display_id}"
+
+
+class UnknownAndroidDisplay(AndroidDisplay):
+    """
+    Fallback display for when the Agent OS is not able to determine the displays.
+    """
+
+    def __init__(self) -> None:
+        super().__init__(0, "Unknown", 0)
+
+    def get_display_id_flag(self) -> str:
+        return ""
+
+    def get_display_unique_id_flag(self) -> str:
+        return ""
```

Comment:
Returning an empty string is a little bit akward.

Can we not allow to return Optional[str] or introduce a `def is_screen_unknown()` function

Reply:
More or less. We need to remove it.